sn_transfers/src/wallet/local_store.rs

```diff
@@ -521,39 +521,6 @@ impl LocalWallet {
     pub fn derive_key(&self, derivation_index: &DerivationIndex) -> DerivedSecretKey {
         self.key.derive_key(derivation_index)
     }
-
-    /// Lookup previous payments and subtract them from the payments we're about to do.
-    /// In essence this will make sure we don't overpay.
-    pub fn take_previous_payments_into_account(
-        &self,
-        payment_map: &mut BTreeMap<XorName, Vec<(MainPubkey, NanoTokens)>>,
-    ) {
```

Comment:
Remove this function, it's a first step in removing repayments, others will need to wait for payment quotes!